setup.py

```diff
@@ -58,8 +58,10 @@ def run(self):
 
         if self.type in ['unit', 'all']:
             cmd.append('python/Ganga/new_tests/Unit')
-            cmd.append('python/Ganga/Core')
-            cmd.append('python/Ganga/Runtime')
+            ## Ganga/Core finds zero tests
+            #cmd.append('python/Ganga/Core')
+            ## Ganga/Runtime imports GangaProgram but causes a vanilla instance to be loaded
+            #cmd.append('python/Ganga/Runtime')
```

Comment:
Where is this vanilla instance being created? The only time this object is instantiated should be in `bin/ganga` and in `GangaUnitTest`.